Scripts/Game/ODIN/Helpers/ODIN_VisibilityHelper.c

```diff
@@ -0,0 +1,31 @@
+/**
+Helper class with staic functions to enable and disable visiblity of units
+*/
+
+class ODIN_VisibilityHelper
+{
+	// Need different function as we have to cast to different parent objects for RPL functionality. Maybe templating could work?
+	static void SetVisibilityCharacter(GenericEntity entity, bool visible)
```

Comment:
I would say we just have one `SetVisibility` method. You just cast it to `SCR_ChimeraCharacter` and if it's not `null`, then call the method and repeat the same for `Vehicle`